package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "build-clean": "rm -rf ./packages/*/build",
     "build": "node ./scripts/build.js",
-    "clean-all": "rm -rf ./node_modules; rm -rf ./packages/*/node_modules; rm -rf ./integration_tests/*/*/node_modules; yarn run build-clean",
+    "clean-all": "rm -rf ./node_modules && rm -rf ./packages/*/node_modules && rm -rf ./integration_tests/*/*/node_modules && yarn run build-clean",
```

Comment:
Normally, you'd use `rimraf`, but since `clean-all` removes `node_modules`, that's not an option.  I'm not sure if there is a cross-platform way to do this.